src/webgpu/shader/README.txt

```diff
@@ -1 +1,6 @@
 Tests for full coverage of the shaders that can be passed to WebGPU.
+
+This folder contains test plans generated by the following tool:
+tool: https://dawn.googlesource.com/tint/+/refs/heads/main/tools/src/cmd/get-test-plan/main.go
+
+How to contribute doc: https://github.com/gpuweb/cts/docs/plan_autogen.md
```

Comment:
would be `https://github.com/gpuweb/cts/blob/main/docs/plan_autogen.md` I think, or use a path relative to the root of the project or relative to this file.